### DISCUSSION TITLE
Increased precision of angle

### DISCUSSION THREAD

**Discussion by eecue** [terjeio/ioSender#223] [Ideas]
Would it be possible to add two or three more decimal places to the angle probing functionality? Or maybe even automate the setting of the dual y offset ganging?  Here’s what I’m trying to do but it’s not very effective with only tenths of degrees:

https://youtu.be/4LfjH58n0YQ

**Comment by terjeio** [comment]
New [edge version ](http://www.io-engineering.com/downloads) uploaded, displays 3 decimal places.

> Or maybe even automate the setting of the dual y offset ganging?

I don't think I want to spend time developing and testing such a feature as I do not have a physical machine to test with. You can do it?
